app/services/queue_listener.rb

```diff
@@ -60,20 +60,6 @@ def cancel_subscription(params)
     end
   end
 
-  def create_donation(params)
-    order = params[:params][:order]
-    action = Action.find_by_id(params[:meta][:action_id])
-    response = client.create_donation(params[:params])
-
-    if action
-      action[:form_data][:ak_resource_id] = response['resource_uri']
-      action.save
-    end
```

Comment:
Can you confirm we don't use `ak_resource_id` anywhere?

Reply:
yeah, checked and we don't use it!